yjit/src/core.rs

```diff
@@ -1774,9 +1773,25 @@ pub struct IseqPayload {
     // Blocks that are invalidated but are not yet deallocated.
     // The code GC will free them later.
     pub dead_blocks: Vec<BlockRef>,
+
+    // Process CPU time when compilation started for this ISEQ
+    pub comp_time_start: f64,
```

Comment:
Since the option specifies a timeout in seconds, we probably don't need all the 64 bits to maintain the accuracy. Could we use something like `u32` that stores seconds elapsed since the process started?